Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -99,6 +100,27 @@ - (void)dealloc
   [_sockets[socketID] send:message];
 }
 
+RCT_EXPORT_METHOD(sendBlob:(NSDictionary *)blob socketID:(nonnull NSNumber *)socketID)
+{
+  RCTBlobManager *blobManager = [[self bridge] moduleForClass:[RCTBlobManager class]];
```

Comment:
@javache okies. let's talk about how to do this on slack? I'd really prefer not to do it in the current PR and do a followup for that since this PR and the follow up PR to this are quite big.

also probably worth noting that in my mind eventually more native modules are going to support blob, (there's an open PR for Networking module), e.g. - maybe image module should be able to render a blob, camera roll module should return blobs etc.